sycl/test-e2e/USM/math.cpp

```diff
@@ -25,7 +25,8 @@ int main() {
       {
         float *Buf = (float *)s::malloc_shared(
             sizeof(float) * 2, myQueue.get_device(), myQueue.get_context());
-        s::malloc_shared(100, myQueue.get_device(), myQueue.get_context());
+        void *ptr =
```

Comment:
Considering that the `ptr` is unused, shouldn't have we just deleted that extra `malloc` call?